kousa/lib/business-logic/room_chat_logic.ex

```diff
@@ -94,4 +98,38 @@ defmodule Kousa.BL.RoomChat do
 
     :ok
   end
+
+  # Delete room chat messages
+  def delete_message(deleter_id, message_id, user_id) do
+
+    case Kousa.Data.Room.get_room_status(deleter_id) do
+      {:creator, room} ->
+        RegUtils.lookup_and_cast(
+          Gen.RoomChat,
+          room.id,
+          {:message_deleted, deleter_id, message_id}
+        )
+
+      {:mod, room} ->
+        if user_id != room.creatorId do
```

Comment:
this allows mods to delete other mods messages. I'm ok with this so we can avoid more DB queries, but can you add a comment about this